### DISCUSSION TITLE
How do I stop fonts.gstatic.com from setting cookies

### DISCUSSION THREAD

**Discussion by BethProedrou** [google/docsy#2041] [Q&A] [answered]
During development I noticed that Privacy Badger and Ghostery were detecting google cookies. 
According to the Privacy Badger fonts.gstatic.com is setting third party cookies. How can I deactivate them in order to be in compliance with the GDPR?

**Comment by fekete-robert** [comment]
I think there are two ways around that. You can either:
- add the fonts you use statically to your project, so the clients download them from your site instead of the cdn, and configure docsy to use that
- add a cookie consent banner script that prevents dropping cookies unless the user has clicked accepting cookies

**Reply by BethProedrou** [reply]
Thank you for the answer. I would prefer the first option. Could you tell me how to do that with Docsy or point me to the documents that explain how to do it?

**Comment by fekete-robert** [comment, marked as answer]
Hi, see this thread: https://github.com/google/docsy/discussions/1928

**Reply by BethProedrou** [reply]
Thank you very much!
